src/natcap/invest/hra.py

```diff
@@ -2584,11 +2584,11 @@ def _get_criteria_dataframe(base_criteria_table_path):
     file_ext = os.path.splitext(base_criteria_table_path)[1].lower()
     if file_ext == '.csv':
         # use sep=None, engine='python' to infer what the separator is
-        criteria_df = pandas.read_csv(base_criteria_table_path, 
-            index_col=0, header=None, sep=None, engine='python')
+        criteria_df = pandas.read_csv(base_criteria_table_path,
+                                      index_col=0, header=None, sep=None, engine='python')

```

Comment:
Does this bust the 80 column mark? Maybe something like this here:
```python
criteria_df = pandas.read_csv(
    base_criteria_table_path, index_col=0, header=None, sep=None, 
    engine='python')
```

Reply:
fixed! I need to find out how to configure autopep8 to move args down to the next line